exiftool.go

```diff
@@ -95,6 +104,24 @@ func (e *Exiftool) Close() error {
 		errs = append(errs, fmt.Errorf("error while closing stdin: %w", err))
 	}
 
+	ch := make(chan struct{})
+	go func() {
+		if e.cmd != nil {
+			if err := e.cmd.Wait(); err != nil {
+				errs = append(errs, fmt.Errorf("error while waiting for exiftool to exit: %w", err))
+			}
+		}
+		ch <- struct{}{}
+		close(ch)
```

Comment:
What if `e.cmd.Wait()` blocks forever ? The timeout will be reached and  `ch`might never be closed.
You perhaps should close it later (after the `select` ?).

Reply:
Shouldn't senders close the channel? If `Wait()` finishes after the timeout, we'd panic trying to send on a closed channel. I think it's better to potentially leak channels and goroutines (if `Wait()` blocks forever) than to potentially panic.

Reply:
You're perfectly right !